src/main/java/cloud/eppo/ufc/dto/FlagConfigResponse.java

```diff
@@ -34,7 +34,12 @@ public Map<String, BanditReference> getBanditReferences() {
     return this.banditReferences;
   }
 
-  public boolean isForServer() {
-    return this.forServer;
+  public Format getFormat() {
+    return format;
+  }
+
+  public enum Format {
+    SERVER,
+    CLIENT
```

Comment:
Is this replacing obfuscation flag? If yes, it seems to be a wrong dichotomy because clients may work with non-obfuscated configs, and servers may work with obfuscated ones.

Reply:
yes, this replaces the flag. 
see https://github.com/Eppo-exp/eppo/pull/10896